README.md

```diff
@@ -1,3 +1,6 @@
+> [!WARNING]
+> This package is unmaintained: Active work is not occuring on this package. For an active fork of the package, see [Format.jl](https://github.com/JuliaString/Format.jl).
```

Comment:
Would you please add a badge from repostatus.org? Probably [unsupported](https://www.repostatus.org/#unsupported) (I would add the markdown code, but I'm on my phone and it's hard to copy the long string

Reply:
Unsupported gives me kinda the wrong vibe. I'd think either `Inactive` or `Abandoned` are more accurate. I think `Abandoned` is maybe the most correct and useful since it signals that if someone wanted, they could try and take it over, but it also sounds kinda harsh.

Reply:
Actually, after reading the description, `Unsupported` does seem like the right descriptor.

Reply:
[](https://www.repostatus.org/#Abandoned):
> Initial development has started, but there has not yet been a stable, usable release; the project has been abandoned and the author(s) do not intend on continuing development.

[Unsupported](https://www.repostatus.org/#unsupported):
> The project has reached a stable, usable state but the author(s) have ceased all work on it. A new maintainer may be desired."

So the latter seems appropriate, by their definition ?
(I do agree that the adjective itself does not convey the idea that it would be very cool if someone stepped in).

Reply:
Considering there are quite a lot of egregious bugs that seriously impact usability, I'd say this package is more abandoned than unsupported.